tests/modules/configs/test_config.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pytest
+import sys
+
+from jsonschema import ValidationError
+
+sys.path.append(os.getcwd())
+
+from ingestation.modules.configs.config import validate_json
+from ingestation.modules.configs.schemas import application, project, operator
+from tests.data.configs import validate_json_passed, validate_json_failed, err_msg
```

Comment:
Such import simply won't work after install. We need to define a new package, update setup.cfg (example below) and adjust developer tools.
```
[options]
package_dir =
    ingestation=src
    ingestation_test=tests
packages =
    ingestation
    ingestation.modules
    ingestation.modules.configs
    ingestation.modules.configs.schemas
    ingestation.modules.configs.templates
    ingestation.modules.global_vars
    ingestation.modules.misc
    ingestation.modules.outputs
    ingestation_test
    ingestation_test.data
    ingestation_test.data.configs
```